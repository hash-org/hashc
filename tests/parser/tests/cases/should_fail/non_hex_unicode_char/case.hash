let k = '\u{g00000}';
