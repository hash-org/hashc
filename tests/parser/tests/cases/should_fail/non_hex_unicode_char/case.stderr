error: Failed to parse
 --> $DIR/case.hash:1:10
1 |   k := '\u{g00000}';
  |            ^ here
  = note: Invalid character escape sequence. Expected `}` after a escape sequence
